### PR TITLE
Augmentation Modifier Description Rework

### PR DESCRIPTION
**Pull request by Kwazygloo**
Augmentation descriptions are consistent with their assigned multipliers. 

Removed all references to "the player's".

Cleaned up wording and condensed sentence structure as appropriate.

Augmentation descriptions were changed based on the following rules:

hacking_mult: "Increases Hacking by number%."
strength_mult: "Increases Strength by number%."
defense_mult: "Increases Defense by number%."
dexterity_mult: "Increases Dexterity by number%."
agility_mult: "Increases Agility by number%."
charisma_mult: "Increases Charisma by number%."

hacking_exp_mult: "Increases Hacking experience gain by number%."
strength_exp_mult: "Increases Strength experience gain by number%."
defense_exp_mult: "Increases Defense experience gain by number%."
dexterity_exp_mult: "Increases Dexterity experience gain by number%."
agility_exp_mult: "Increases Agility experience gain by number%."
charisma_exp_mult: "Increases Charisma experience gain by number%."

hacking_chance_mult: "Increases Hacking success chance by number%."
hacking_speed_mult:  "Increases Hacking speed by number%."
hacking_money_mult: "Increases money gained from Hacking by number%."
hacking_grow_mult: "Improves grow() by number%."

faction_rep_mult: "Increases reputation gain from factions by number%."
company_rep_mult: "Increases reputation gain from companies by number%."

crime_money_mult: "Increases money gained from committing crimes by number%."
crime_success_mult: "Increases crime success rate by number%."
work_money_mult: "Increases money gained from working by number%."

hacknet_node_money_mult: "Increases money produced from Hacknet Nodes by number%."
hacknet_node_purchase_cost_mult: "Decreases the purchase cost of Hacknet Nodes by number%."
hacknet_node_level_cost_mult:  "Decreases the upgrade cost of Hacknet Nodes by number%.",

bladeburner_max_stamina_mult: "Increases Bladeburner Stamina by number%."
bladeburner_stamina_gain_mult: "Increases Bladeburner Stamina gain by number%."
bladeburner_analysis_mult: "Increases Bladeburner Field Analysis effectiveness by number%."
bladeburner_success_chance_mult: "Increases success chance in Bladeburner Contracts and Operations by number%."

// If an augment provides the same % bonus to all multipliers in any group

[
hacking_mult:
strength_mult:
defense_mult:
dexterity_mult:
agility_mult:
charisma_mult:
]

"Increases all stats by number%."

[
hacking_exp_mult:
strength_exp_mult:
defense_exp_mult:
dexterity_exp_mult:
agility_exp_mult:
charisma_exp_mult:
]

"Increases experience gain for all stats by number%."

[
strength_mult:
defense_mult:
dexterity_mult:
agility_mult:
]

"Increases all combat stats by number%."

[
strength_exp_mult:
defense_exp_mult:
dexterity_exp_mult:
agility_exp_mult:
]

"Increases experience gain for all combat stats by number%."

[
faction_rep_mult:
company_rep_mult:
]

"Increases reputation gain from factions and companies by number%."

// Unique

BladesSimulacrum:

"Allows the simultaneous performance of Bladeburner actions and other actions."

CashRoot:

"Provides {Money(1e6)} after a reset."
"Provides BruteSSH.exe after a reset."

Neurolink:

"Provides FTPCrack.exe and relaySMTP.exe after a reset."

NeuroFluxGovernor:

"Each level increases all multipliers by 1%."

### CODE REVIEW

**Comment by MageKing17** [comment]
It seems to me that if you have this rigid set of rules for turning multipliers into descriptions, they could be done programmatically instead of needing to be hardcoded.

**Comment by Kwazygloo** [comment]
> It seems to me that if you have this rigid set of rules for turning multipliers into descriptions, they could be done programmatically instead of needing to be hardcoded.

Makes sense, but I already did the work.

File: src/Augmentation/AugmentationHelpers.jsx [hunk 1]
```diff
@@ -1097,8 +1083,7 @@ function initAugmentations() {
              "monitors and regulates nervous impulses coming to and from the spinal column, " +
              "essentially 'governing' the body. By doing so, it improves the functionality of the " +
              "body's nervous system.<br><br>" +
-             "This is a special augmentation because it can be leveled up infinitely. Each level of this augmentation " +
-             "increases ALL of the player's multipliers by 1%.",
+             "Each level increases all multipliers by 1%.",
```
Comment (danielyxie) [inline src/Augmentation/AugmentationHelpers.jsx:1086]: I would keep the part about it being possible for this Augmentation to be leveled up infinitely, as that is important information

File: src/Augmentation/AugmentationHelpers.jsx [hunk 2]
```diff
@@ -1252,11 +1235,10 @@ function initAugmentations() {
         info:"This is a more advanced version of the LuminCloaking-V2 augmentation. This skin implant " +
              "reinforces the skin with highly-advanced synthetic cells. These " +
              "cells, when powered, are capable of not only bending light but also of bending heat, " +
-             "making the user more resilient as well as stealthy.<br><br>" +
-             "This augmentation:<br>" +
-             "Increases the player's agility by 10%.<br>" +
-             "Increases the player's defense by 10%.<br>" +
-             "Increases the amount of money the player gains from crimes by 25%.",
+             "making the user more resilient as well as stealthy.<br><br>" ++
```
Comment (danielyxie) [inline src/Augmentation/AugmentationHelpers.jsx:1238]: Extra plus sign at the end here

File: src/Augmentation/AugmentationHelpers.jsx [hunk 3]
```diff
@@ -1951,9 +1919,8 @@ function initAugmentations() {
                  "extracellular matrix (ECM). This improves the ECM's ability to " +
                  "structurally support the body and grants heightened strength and " +
                  "durability.<br><br>" +
-                 "This augmentation:<br>" +
-                 "Increases the player's experience gain rate for all combat stats by 5%.<br>" +
-                 "Increases the player's Bladeburner max stamina by 10%.",
+                 "Increases experience gain for all combat stats by 5%.<br>" +
+                 "Increases Bladeburner Stamina by 10%.",
```
Comment (danielyxie) [inline src/Augmentation/AugmentationHelpers.jsx:1923]: I'd add the word "max" back in here to be clear: `Bladeburner max stamina`. Same for all of the others

File: src/Augmentation/AugmentationHelpers.jsx [hunk 4]
```diff
@@ -1970,10 +1937,9 @@ function initAugmentations() {
                  "during the Synthoid Uprising. The organic musculature of the human foot " +
                  "is enhanced with flexible carbon nanotube matrices that are controlled by " +
                  "intelligent servo-motors.<br><br>" +
-                 "This augmentation:<br>" +
-                 "Increases the player's agility by 5%.<br>" +
-                 "Increases the player's Bladeburner max stamina by 5%.<br>" +
-                 "Increases the player's Bladeburner stamina gain rate by 5%.<br>",
+                 "Increases Agility by 5%.<br>" +
+                 "Increases Bladeburner Stamina by 5%.<br>" +
+                 "Increases Bladeburner Stamina gain by 5%.<br>",
```
Comment (danielyxie) [inline src/Augmentation/AugmentationHelpers.jsx:1942]: We don't need this `<br>` at the end here

**Review by danielyxie — CHANGES_REQUESTED** [review]

**Comment by hydroflame** [comment]
These changes were included in some other form.

**Pull request closed without merging**